official/resnet/keras/keras_cifar_benchmark.py

```diff
@@ -147,38 +147,69 @@ def benchmark_1_gpu_no_dist_strat_run_eagerly(self):
     FLAGS.distribution_strategy = 'off'
     self._run_and_report_benchmark()
 
-  def benchmark_2_gpu(self):
-    """Test keras based model with eager and distribution strategies."""
+  def benchmark_graph_1_gpu_no_dist_strat(self):
+    """Test keras based model with Keras fit but not distribution strategies."""
     self._setup()
-    FLAGS.num_gpus = 2
+    FLAGS.distribution_strategy = 'off'
+    FLAGS.num_gpus = 1
     FLAGS.data_dir = self.data_dir
     FLAGS.batch_size = 128
     FLAGS.train_epochs = 182
-    FLAGS.model_dir = self._get_model_dir('benchmark_2_gpu')
+    FLAGS.model_dir = self._get_model_dir('benchmark_graph_1_gpu_no_dist_strat')
+    FLAGS.dtype = 'fp32'
+    self._run_and_report_benchmark()
+
+  def benchmark_1_gpu_force_dist_strat(self):
+    """No dist strat but forced ds tf.compile path."""
+    self._setup()
+    FLAGS.distribution_strategy = 'off'
+    FLAGS.num_gpus = 1
+    FLAGS.data_dir = self.data_dir
+    FLAGS.batch_size = 128
+    FLAGS.train_epochs = 182
+    FLAGS.model_dir = self._get_model_dir('benchmark_1_gpu_force_dist_strat')
     FLAGS.dtype = 'fp32'
     FLAGS.enable_eager = True
+    FLAGS.distribution_strategy = 'off'
+    FLAGS.force_run_distributed = True
     self._run_and_report_benchmark()
 
-  def benchmark_graph_2_gpu(self):
-    """Test keras based model with Keras fit and distribution strategies."""
+  def benchmark_1_gpu_force_dist_strat_run_eagerly(self):
```

Comment:
I am not sure whether the new "force_run_distributed" has any impact when "run_eagerly=True". My understanding was that it was only useful for the default case which uses tf.function/func-graph etc. Worth confirming otherwise we may be running 2 identical benchmarks

Reply:
Just checked with @isaprykin , it seems like it will have a different codepath, so probably fine to benchmark both versions then.